stable/spartakus/templates/deployment.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    heritage: "{{ .Release.Service }}"
+    release: "{{ .Release.Name }}"
+  name: {{ template "fullname" . }}
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: {{ template "fullname" . }}
+        release: "{{ .Release.Name }}"
+    spec:
+      containers:
+        - name: {{ template "fullname" . }}
+          image: "{{ .Values.image }}"
+          imagePullPolicy: {{ default "" .Values.imagePullPolicy | quote }}
+          args:
+            - volunteer
+            - --cluster-id={{ uuidv4 | default .Values.uuid | quote }}
+          resources:
+{{ toYaml .Values.resources | indent 12 }}
```

Comment:
I believe this way the limits and requests can both be passed in values.yaml. If the ask is to add an arbitrary limit to values.yaml, I would just comment it out to leave it as an example.